src/test/scala/com/fulcrumgenomics/vcf/AssessPhasingTest.scala

```diff
@@ -205,6 +207,10 @@ class AssessPhasingTest extends ErrorLogLevel {
     val header = new SAMFileHeader
     header.setSequenceDictionary(Header.getSequenceDictionary)
     val intervalList = new IntervalList(header)
+    // TODO: BUG!!! These variables are never passed down into the interval list.
+    locally { val _ = start }
+    locally { val _ = end }
+    // TODO: BUG!!! This is meant to be start and end, but that is not happening.
```

Comment:
> [!WARNING]
> This concerns me. It is a bug in the test. When you replace `start` and `end` in the interval that is added to the interval instead of the default `1` and `10`, many tests start to fail. I don't know if the implementation of `AssessPhasing` is wrong or if the tests that were written were copy-pasted from the outcome of the bugged test info.

I will open an issue so someone can solve in a future PR.

Reply:
Yes, please open an issue, since there's a bug here in that case.

Reply:
https://github.com/fulcrumgenomics/fgbio/issues/1050